buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -83,6 +83,19 @@ class BuildPlugin implements Plugin<Project> {
 
     /** Adds additional manifest info to jars */
     static void configureJarManifest(Project project) {
+        project.scminfo {
+            def dotgit = "$project.rootProject.projectDir/.git/"
```

Comment:
Hrm, also I think we need to do a traversal here, going up from the current project's project dir. We can't assume rootProject is the git root (for example, if elasticsearch is attached to another project with the project-attachment-plugin).